internal/pkg/template/templates/environment/cf.yml

```diff
@@ -390,6 +470,26 @@ Outputs:
     Value: !Ref DefaultHTTPTargetGroup
     Export:
       Name: !Sub ${AWS::StackName}-DefaultHTTPTargetGroup
+  InternalLoadBalancerDNSName:
+    Condition: CreateInternalALB
+    Value: !GetAtt InternalLoadBalancer.DNSName
+    Export:
+      Name: !Sub ${AWS::StackName}-InternalLoadBalancerDNS
+  InternalLoadBalancerFullName:
```

Comment:
Can I ask for a favor? 🥺  maybe let's remove these outputs (for now) and only add them when you make modification to the backend service CFN template so that we can guarantee that only outputs that we need get exported